irohad/ordering/impl/on_demand_ordering_gate.cpp

```diff
@@ -49,8 +51,8 @@ void OnDemandOrderingGate::propagateBatch(
   }
 
   // TODO iceseer 14.01.21 IR-959 Refactor to avoid copying.
```

Comment:
What about its: https://jira.hyperledger.org/browse/IR-959 maybe it can be solved now (after one year after reporting)?

Reply:
:) Not now, but soon :)
https://github.com/hyperledger/iroha/blob/feature/mst_os_unite/irohad/ordering/impl/on_demand_ordering_gate.cpp#L51